### PR TITLE
FOR-7-fix-task-explorer

### PR DESCRIPTION
**Pull request by RoryGolledge**
# Description

Upon changing to Task Explorer, a 405 error occurred due to a bad GET request. This is fixed.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Various methods of manual navigation. Going from different tabs to Task Explorer, refreshing on Task Explorer etc.

## Testing Checklist:

- [x] Tested in latest Chrome
- [x] Tested in latest Safari
- [ ] Tested in latest Firefox

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from @macite and @jakerenzella on the Pull Request

### CODE REVIEW

**Comment by jakerenzella** [comment]
@RoryGolledge This PR needs to be made against the https://github.com/doubtfire-lms/doubtfire-web/tree/FOR-1-task-inbox-redesign branch

File: src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.component.ts [hunk 1]
```diff
@@ -294,6 +296,9 @@ export class StaffTaskListComponent implements OnInit, OnChanges {
 
   // Callback to refresh data from the task source
   private refreshData() {
+    // if in task def mode, and filters is null
+
+
```
Comment (jakerenzella) [inline src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.component.ts]: I don't think we need this comment

**Review by jakerenzella — CHANGES_REQUESTED** [review]
PR looks good @RoryGolledge  just one (very) small change!

**Comment by jakerenzella** [comment]
Also PR title should be something like: FOR-7-fix-task-explorer

**Comment by RoryGolledge** [comment]
Perfect. Applied feedback

**Comment by jakerenzella** [comment]
lgtm @RoryGolledge  🥳

**Pull request merged**